TM1py/Services/ElementService.py

```diff
@@ -135,6 +180,22 @@ def get_number_of_leaf_elements(self, dimension_name: str, hierarchy_name: str,
         response = self._rest.GET(url, **kwargs)
         return int(response.text)
 
+    def get_number_of_numeric_elements(self, dimension_name: str, hierarchy_name: str, **kwargs) -> int:
+        url = format_url(
+            "/api/v1/Dimensions('{}')/Hierarchies('{}')/Elements/$count?$filter=Type eq 1",
+            dimension_name,
+            hierarchy_name)
+        response = self._rest.GET(url, **kwargs)
+        return int(response.text)
+
+    def get_number_of_string_elements(self, dimension_name: str, hierarchy_name: str, **kwargs) -> int:
+        url = format_url(
+            "/api/v1/Dimensions('{}')/Hierarchies('{}')/Elements/$count?$filter=Type eq 2",
+            dimension_name,
+            hierarchy_name)
+        response = self._rest.GET(url, **kwargs)
```

Comment:
Did you verify that the response contains only the number and no actual elements in the JSON?

Reply:
Yep, verified only number returned in response. This is copy of the `get_number_of_leaf_elements` function with the filter value being modified.